src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -33,11 +34,14 @@ final class CartContext implements Context
     private $cartsClient;
 
     /** @var ApiClientInterface */
-    private $productsClient;
+    private $productsShopClient;
```

Comment:
This is shop context, so IMO you could leave it as it was, especially that `$cartsClient` is without `Shop` part